### PR TITLE
[OKTA-190165] Policy API doc: fixing code example for creating a rule

### PR DESCRIPTION
**Pull request by motikagan-okta**
## Description:
- Policy API doc: fixing code example for creating a rule, adding name parameter to rules object table.

### Resolves:

* [OKTA-190165](https://oktainc.atlassian.net/browse/OKTA-190165)

### CODE REVIEW

**Comment by motikagan-okta** [comment]
screen shot 2018-11-07 at 4 38 02 pm

**Comment by motikagan-okta** [comment]
screen shot 2018-11-07 at 4 38 48 pm

**Comment by motikagan-okta** [comment]
@mehrshadrafiei-okta, I added screenshots of the 'Create a rule' example code and of the 'Rules object' table--those are the two sections of the page that are changed.

File: _source/_docs/api/resources/policy.md [hunk 1]
```diff
@@ -325,7 +325,7 @@ curl -v -X POST \
     "network": {
       "connection": "ZONE",
       "include": [
-        "nzowdja2YRaQmOQYp0g3"
```
Comment (royalchan-okta) [inline _source/_docs/api/resources/policy.md]: nit: we generally like to keep IDs consistent across examples on a single page, so I would revert this change unless you have a good reason for it

File: _source/_docs/api/resources/policy.md [hunk 2]
```diff
@@ -336,7 +336,8 @@ curl -v -X POST \
     "signon": {
       "access": "ALLOW"
     }
-  }' \
```
Comment (royalchan-okta) [inline _source/_docs/api/resources/policy.md]: 👍

File: _source/_docs/api/resources/policy.md (hunk 1 above)
Comment (motikagan-okta) [reply _source/_docs/api/resources/policy.md]: Thanks, Royal. Have reverted the change to the network ID in the example.
